.github/workflows/reports.yml

```
@@ -428,11 +442,11 @@ jobs:
 
       - name: Parse compilation report
         id: compilation_report
-        uses: noir-lang/noir-bench-report@e408e131e96c3615b4f820d7d642360fb4d6e2f4
+        uses: noir-lang/noir-bench-report@e0a9b8e6bad698c2b1d55819fef45324a9a2f12c
```

```suggestion
        uses: noir-lang/noir-bench-report@6ba151d7795042c4ff51864fbeb13c0a6a79246c
```

Can you update the other references to this action so they're in sync?